catalogs/catalog-jdbc-oceanbase/src/main/java/org/apache/gravitino/catalog/oceanbase/operation/OceanBaseTableOperations.java

```diff
@@ -58,10 +93,130 @@ protected String generateCreateTableSql(
       Distribution distribution,
       Index[] indexes) {
     if (ArrayUtils.isNotEmpty(partitioning)) {
-      throw new UnsupportedOperationException("Currently not support Partition tables.");
+      throw new UnsupportedOperationException(
+          "Currently we do not support Partitioning in oceanbase");
+    }
```

Comment:
Also do not support distribution and indexes, right?

Reply:
Supports indexes, but not support distribution.
A check will be added for OceanBase does not support distribution.